packages/api/src/sdk-bundle-api/sdk-to-bundle/websocket-data.ts

```diff
@@ -37,10 +37,24 @@ export interface WebSocketConnectionOpenedEvent
   type: "opened";
 }
 
+export interface EncodedArrayBuffer {
+  binaryType: "arraybuffer";
+  /** Base 64 encoded binary from an array buffer */
+  encodedData: string;
+}
+
+export interface EncodedBlob {
+  binaryType: "blob";
+  /** Base 64 encoded binary from a blob */
+  encodedData: string;
```

Comment:
Fun fact I learnt today: although base64 uses 33% extra storage, if the network traffic is gzipped the storage penalty almost disappears, leaving only a 2.5-5% penalty (those wasted 2 bits for every 8 get compressed away anyway).

Reply:
Interesting. Do we gzip session data when sending it to / loading it from S3?